### PR TITLE
Add ellucian student connector

### PR DESCRIPTION
**Pull request by SkNuwanTissera**
# Description

Ellucian is a higher education platform connecting people, processes and applications across the institution to power coordinated programs designed for student success. Ellucian Student API provides the capability to retrieve details about students in Colleague.

Fixes https://github.com/wso2-enterprise/choreo/issues/15712

## Type of change
- [X] New feature (non-breaking change which adds functionality)

**Test Configuration**:
* Ballerina Version: Ballerina 2201.1.1
* Operating System: Unix
* Java SDK: Java11

# Checklist:

### Changes to OpenAPI definition

- [X] Modified `info` section of the OpenAPI definition file - [Example](https://github.com/ballerina-platform/ballerina-extended-library/discussions/74)

    - `description` - what the connector is about 
    - `x-ballerina-init-description` OpenAPI extension - connector initialization details 
    - `x-ballerina-display` OpenAPI extension - connector name and path to the connector icon 

- [x] Added proper description each API Key, if API Key authentication is defined in OpenAPI definition file 

### Changes to connector module

- [X] Added license header at the top of each .bal file. 
- [X] Added Package.md as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/77)
- [X] Added Module.md for the module of the  connector as per the guide [here](https://github.com/ballerina-platform/ballerina-extended-library/discussions/78)
- [X] Added connector icon to the root of the connector. Icon needs to be a `png` of 200x200 px size with name `icon.png`
- [X] Added Ballerina.toml file with following information. For keywords refer to [guide](https://github.com/ballerina-platform/ballerina-extended-library/discussions/72)

### Verifying connector module

- [X] Compiled the connector successfully with the targeted ballerina version. 
- [X] Conducted smoke tests on the connector (Recommended when OpenAPI definition is obtain from an unofficial source)

### Security checks
 - [X] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? 
 - [X] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: openapi/ellucian.student/Ballerina.toml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+org = "ballerinax"
+name = "ellucian.student"
+icon = "icon.png"
+distribution = "2201.1.1"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.0"
+authors = ["Ballerina"]
+
+[build-options]
+observabilityIncluded = true
```
Comment (RolandHewage) [inline openapi/ellucian.student/Ballerina.toml:13]: Let's add a newline at the end of the file.

**Review by RolandHewage — APPROVED** [review]
Just a minor modification is requested. Rest is fine. LGTM.

**Comment by SkNuwanTissera** [comment]
Let's keep this PR OPEN until this is tested in DEV.

File: openapi/ellucian.student/Ballerina.toml (hunk 1 above)
Comment (SkNuwanTissera) [reply openapi/ellucian.student/Ballerina.toml]: Updated. Thanks

**Comment by NipunaRanasinghe** [comment]
Closing as per the offline discussion I had with @LakshanSS, as we are working on cleaning up outdated PRs in the Ballerina platform repositories and this PR is not needed anymore.

**Pull request closed without merging**